server/src/main/java/io/unitycatalog/server/persist/dao/StorageCredentialDAO.java

```diff
@@ -0,0 +1,128 @@
+package io.unitycatalog.server.persist.dao;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.unitycatalog.server.model.AwsIamRoleResponse;
+import io.unitycatalog.server.model.AzureServicePrincipal;
+import io.unitycatalog.server.model.StorageCredentialInfo;
+import jakarta.persistence.*;
+import java.time.Instant;
+import java.util.Date;
+import java.util.UUID;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+@Entity
+@Table(name = "uc_storage_credential")
```

Comment:
are all other tables names starting with "uc_"?
and are they singular or plural? that is, `...credential` or `...credentials`?